transmart-rest-api/src/main/groovy/org/transmartproject/rest/protobuf/SerializableProperties.groovy

```diff
@@ -8,11 +8,11 @@ class SerializableProperties {
                                                         "age", "race", "maritalStatus",
                                                         "religion", "sourcesystemCd", "sexCd"]
     public static final Set<String> CONCEPT = ["conceptPath", "conceptCode"]
-    public static final Set<String> STUDY = ["studyId", "bioExperimentId", "secureObjectToken"]
-    public static final Set<String> TRIAL_VISIT = ["id", "relTimeUnit", "relTime", "relTimeLabel"]
+    public static final Set<String> STUDY = ["studyId"]
+    public static final Set<String> TRIAL_VISIT = ["relTimeUnit", "relTime", "relTimeLabel"]
     public static final Set<String> START_DATE = ["startDate"]
     public static final Set<String> END_DATE = ["endDate"]
-    public static final Set<String> VISIT = ["encounterNum", "patient", "activeStatusCd", "startDate", "endDate", "inoutCd", "locationCd"]
+    public static final Set<String> VISIT = ["patientInTrialId", "activeStatusCd", "startDate", "endDate", "inoutCd", "locationCd"]
```

Comment:
Please add `encounterNum` as well. Currently, this reverts [Fix for TMPDEV-159](https://github.com/thehyve/transmart-upgrade/commit/e60568e1c186cccf57cd6e9eb9952fac01588525)